code/modules/mob/interactive.dm

```diff
@@ -81,6 +82,82 @@
 	var/traitorScale = 0 // our ability as a traitor
 	var/traitorType = 0
 
+/*	Debug Verb to easily display the SNPCs stats.
```

Comment:
Why comment this out? You should make this a hidden verb that requires debug verbs to be enabled to appear.